src/Parser/PotreeBinParser.js

```diff
@@ -1,53 +1,122 @@
 import * as THREE from 'three';
 import PointsMaterial from '../Renderer/PointsMaterial';
 
+// See the different constants holding ordinal, name, numElements, byteSize in PointAttributes.cpp in PotreeConverter
+// elementByteSize is byteSize / numElements
+const POINT_ATTTRIBUTES = {
+    POSITION_CARTESIAN: {
+        numElements: 3,
+        arrayType: Float32Array,
+        attributeName: 'position',
+    },
+    COLOR_PACKED: {
+        numElements: 4,
+        arrayType: Uint8Array,
+        attributeName: 'color',
+        normalized: true,
+    },
+    INTENSITY: {
+        numElements: 1,
+        numByte: 2,
+        arrayType: Float32Array,
+        attributeName: 'intensity',
+        normalized: true,
+    },
+    CLASSIFICATION: {
+        numElements: 1,
+        arrayType: Uint8Array,
+        attributeName: 'classification',
+    },
+    // Note: at the time of writing, PotreeConverter will only generate normals in Oct16 format
+    // see PotreeConverter.cpp:121
+    // keeping all the values to stay iso to PointAttributes.cpp
```

Comment:
Actually keeping all the values to be able to read clouds converted with old version. I'm editing that.